static/app/views/performance/transactionSummary/transactionEvents/index.tsx

```diff
@@ -0,0 +1,181 @@
+import {Component} from 'react';
+import {browserHistory, WithRouterProps} from 'react-router';
+import {Location} from 'history';
+
+import Feature from 'app/components/acl/feature';
+import Alert from 'app/components/alert';
+import LightWeightNoProjectMessage from 'app/components/lightWeightNoProjectMessage';
+import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
+import SentryDocumentTitle from 'app/components/sentryDocumentTitle';
+import {t} from 'app/locale';
+import {GlobalSelection, Organization, Project} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {
+  isAggregateField,
+  SPAN_OP_BREAKDOWN_FIELDS,
+  SPAN_OP_RELATIVE_BREAKDOWN_FIELD,
+} from 'app/utils/discover/fields';
+import {decodeScalar} from 'app/utils/queryString';
+import {stringifyQueryObject, tokenizeSearch} from 'app/utils/tokenizeSearch';
+import withGlobalSelection from 'app/utils/withGlobalSelection';
+import withOrganization from 'app/utils/withOrganization';
+import withProjects from 'app/utils/withProjects';
+
+import {getTransactionName} from '../../utils';
+import {decodeFilterFromLocation, SpanOperationBreakdownFilter} from '../filter';
+
+import EventsPageContent from './content';
+
+type Props = {
+  location: Location;
+  organization: Organization;
+  projects: Project[];
+  selection: GlobalSelection;
+} & Pick<WithRouterProps, 'router'>;
+
+type State = {
+  spanOperationBreakdownFilter: SpanOperationBreakdownFilter;
+  eventView: EventView | undefined;
+};
+
+class TransactionEvents extends Component<Props> {
```

Comment:
```suggestion
class TransactionSummary extends Component<Props, State> {
```

Reply:
Thanks Shruthi! I've added the `, State`